packages/twenty-server/src/database/typeorm/core/migrations/1715593226719-updateInconsistentUserConstraint.ts

```diff
@@ -0,0 +1,43 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class UpdateInconsistentUserConstraint1715593226719
+  implements MigrationInterface
+{
+  name = 'UpdateInconsistentUserConstraint1715593226719';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `ALTER TABLE "core"."user" DROP CONSTRAINT "FK_2ec910029395fa7655621c88908"`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "core"."billingSubscription" ALTER COLUMN "status" TYPE text`,
```

Comment:
It seems we added "type: 'text'" to the entity file a few weeks ago, this shouldn't create any issue. We should probably use enum in the end though!